consult.el

```diff
@@ -4680,13 +4747,12 @@ Take the variables `grep-find-ignored-directories' and
 BUILDER is the command line builder function.
 PROMPT is the prompt string.
 INITIAL is inital input."
-  (let* ((prompt-dir (consult--directory-prompt prompt dir))
-         (default-directory (cdr prompt-dir)))
+  (let* ((default-directory dir))
```

Comment:
consult--directory-prompt should still be called here, see my comment https://github.com/minad/consult/pull/749/files#r1116595189.

Reply:
Should be resolved in next push.